rott/rt_datadir.c

```diff
@@ -68,13 +68,15 @@ char *GetPrefDir (void)
 
         M_MakeDirectory(dir);
 
-#if !(SHAREWARE == 1)
         result = dir;
+#if !(SHAREWARE == 1)
         dir = M_StringJoin(result, "darkwar", PATH_SEP_STR, NULL);
+#else
+        dir = M_StringJoin(result, "huntbgin", PATH_SEP_STR, NULL);
```

Comment:
Do we already know the name of the WAD at this point and can use this, instead of hard-coding it based on compile time options?

Reply:
The WAD names themselves are still hardcoded based on the same compile time option, so I don't know if it matters here. I think it *should* be changed if we ever do a bigger update to how the game/mod mounting & loading works, but I don't know if now's the time.

Reply:
Okay, thanks!